cmd/controller-manager/main.go

```diff
@@ -165,6 +166,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx := ctrl.SetupSignalHandler()
```

Comment:
Can we move the field indexer setup right before the reconciler setup.

Reply:
Done